lib/broadway_kafka/brod_client.ex

```diff
@@ -297,15 +297,25 @@ defmodule BroadwayKafka.BrodClient do
   defp validate_option(:sasl, value = {:callback, _callback_module, _opts}),
     do: {:ok, value}
 
+  defp validate_option(:sasl, {mechanism, username, password} = value)
+       when mechanism in [:plain, :scram_sha_256, :scram_sha_512] and
+              is_binary(username) and
+              is_binary(password) do
+    {:ok, value}
+  end
+
+  defp validate_option(:sasl, {mechanism, path} = value)
+       when mechanism in [:plain, :scram_sha_256, :scram_sha_512] and
+              is_binary(path) do
```

Comment:
To they actually allow the path to be a binary or must it be an Erlang string (i.e. Elixir charlist?)?

Reply:
We have it working as binary, I'm not sure if it allows a charlist, do you want me to check? I'm following the same typing approach as the existing username/password

Reply:
If it works as a binary, that's all we need.